src/daemon/rpmostreed-daemon.c

```diff
@@ -415,7 +415,8 @@ update_status (RpmostreedDaemon *self)
         have_active_txn = TRUE;
     }
 
-  currently_idle = !have_active_txn && n_clients == 0;
+  if (!getenv ("RPMOSTREE_DEBUG_DISABLE_DAEMON_IDLE_EXIT"))
```

Comment:
I'm cool with this for now, but down the line I think we should use `g_parse_debug_string`.